rust/oak/src/lib.rs

```diff
@@ -261,12 +281,27 @@ pub trait OakNode {
 }
 
 /// Perform an event loop invoking the given node.
-pub fn event_loop<T: OakNode>(mut node: T) -> i32 {
-    info!("start event loop for node");
-    set_panic_hook();
-
+pub fn event_loop<T: OakNode>(node: T) -> i32 {
```

Comment:
We should remove this method once all examples are migrated to use `event_loop_with`?